### ISSUE TITLE
Blurry placeholder performance: (Optionally) use sharp

### ISSUE THREAD

**Issue by cramforce** [ampproject/amp-toolbox#1119]
[sharp](https://www.npmjs.com/package/sharp) should be a much faster image manipulation library. Would be nice if it could be the default, but otherwise making it optional would also be work.

**Comment by sebastianbenz** [comment]
Yeah, we definitely should switch. This has been on my todo list for a while. The current rendering performance is pretty bad.
